Hunting Queries/BehaviorAnalytics/Anomalous Role Assignment.yml

```diff
@@ -0,0 +1,30 @@
+id: 5aa5083c-1de6-42bb-a128-2ec2aba1de39
+name: Anomalous Role Assignment
+description: |
+  'All high Blast Radius users performing Add member to priveleged role, or ones that add users for the first time.'
+requiredDataConnectors:
+  - connectorId: BehaviorAnalytics
+    dataTypes:
+       - BehaviorAnalytics
+       - AuditLogs
+tactics:
+  - Persistence
+relevantTechniques:
+  - T1098
+query: |
+
+ let critical = dynamic(['9b895d92-2cd3-44c7-9d02-a6ac2d5ea5c3','c4e39bd9-1100-46d3-8c65-fb160da0071f','158c047a-c907-4556-b7ef-446551a6b5f7','62e90394-69f5-4237-9190-012177145e10','d29b2b05-8046-44ba-8758-1e26182fcf32','729827e3-9c14-49f7-bb1b-9608f156bbb8','966707d0-3269-4727-9be2-8c3a10f19b9d','194ae4cb-b126-40b2-bd5b-6091b380977d','fe930be7-5e62-47db-91af-98c3a49a38b1']);

```

Comment:
The comments you added in Anomalous Account Manipulation.yaml should be added here also.